liqd_product/apps/actions/assets/timestamps.js

```diff
@@ -11,7 +11,7 @@ $(function () {
     let timeagoInstance = timeago()
 
     if ((new Date() - datetime) < sevenDays) {
-      e.textContent = timeagoInstance.format(datetime, 'de')
+      e.textContent = timeagoInstance.format(datetime, 'en')
```

Comment:
Hmm, I am not sure, we should just change it here. I guess it also needs to depend on the locale?!